src/environments/environment.staging.ts

```diff
@@ -0,0 +1,4 @@
+import { AppConfig as ProdAppConfig } from './environment.prod';
+
+ProdAppConfig.clientId = '54b9dcc49069dc2f018e';
```

Comment:
Just a question @seanlowjk @anubh-v ,
Not really important I think but would it be better practice to deconstruct the `ProdAppConfig` and then change the variable instead of mutating the imported object directly? So something like this?

```typescript
export const AppConfig = {
  ...ProdAppConfig,
  clientId = '54b9dcc49069dc2f018e'
}
```

Reply:
Yes, it wold be better practice to do so actually. 
Let me fix that shortly!